faqs/mkr-token.md

```diff
@@ -28,9 +28,9 @@ Another important responsibility for MKR holders is to communicate publicly abou
 
 ## What voting rights do MKR holders have?
 
-MKR holders have the ability to enact technical changes to the Maker Protocol itself and also to ratify decisions on-chain about non-technical matters. Whether that be a change to the system parameters, or a decision to accept a new governing philosophy, MKR voters have ultimate control. To learn more about governance in the Maker Protocol visit the [Governance FAQ](faqs/governance.md).
+MKR holders have the ability to enact technical changes to the Maker Protocol itself and also to ratify decisions on-chain about non-technical matters. Whether that be a change to the system parameters, or a decision to accept a new governing philosophy, MKR voters have ultimate control. To learn more about governance in the Maker Protocol visit the [Governance FAQ](scd-faqs/scd-faqs/governance.md).
```

Comment:
> To learn more about governance in the Maker Protocol visit the [Governance FAQ](scd-faqs/scd-faqs/governance.md).

this link still goes to a 404

Reply:
ah I accidentally doubled the "scd-faqs"

will push fix today